### PR TITLE
fix bug to running pipeline example

### PR DESCRIPTION
**Pull request by goldwinner**
Please verify the part I modified.It spent me about two days to figured it out the root cause

### CODE REVIEW

**Comment by goldwinner** [comment]
> @googlebot I signed it!

@googlebot I signed it!

File: content/en/docs/azure/azureEndtoEnd.md [hunk 1]
```diff
@@ -117,6 +117,13 @@ To create an app registration:
     alt="Client secret location"
     class="mt-3 mb-3 p-3 border border-info rounded">
 
+### Add "reader" permission for subscription first
+
+1. search for and select **Subscriptions**, or select **Subscriptions** on the **Home** page.
```
Comment (berndverst) [inline content/en/docs/azure/azureEndtoEnd.md:122]: NIT: Start with capital letter please.

File: content/en/docs/azure/azureEndtoEnd.md [hunk 2]
```diff
@@ -58,7 +58,7 @@ To create an ML workspace:
 
 1. Go to [the Azure portal](https://portal.azure.com) and click on your resource group.
 2. Select the **add a new resource** option.
-3. Search for **Machine Learning Studio Workspace** and use the default options, taking note of the name you decide for it.
+3. Search for **Machine Learning** and use the default options, taking note of the name you decide for it.
```
Comment (joeliedtke) [inline content/en/docs/azure/azureEndtoEnd.md:61]: Recommendation: split this into two sentences. For example:

Search for **Machine Learning** and use the default options. Take note of the name you choose for this resource.

File: content/en/docs/azure/azureEndtoEnd.md [hunk 3]
```diff
@@ -117,6 +117,13 @@ To create an app registration:
     alt="Client secret location"
     class="mt-3 mb-3 p-3 border border-info rounded">
 
+### Add "reader" permission for subscription first
+
+1. search for and select **Subscriptions**, or select **Subscriptions** on the **Home** page.
+2. Select **Access control (IAM)** from the sidebar. Select Add a role assignment.
```
Comment (joeliedtke) [inline content/en/docs/azure/azureEndtoEnd.md:123]: Suggestion: Split each instruction onto its own line. For example:

2. Select **Access control (IAM)** from the sidebar. 
3. Select Add a role assignment.

File: content/en/docs/azure/azureEndtoEnd.md (hunk 1 above)
Comment (joeliedtke) [reply content/en/docs/azure/azureEndtoEnd.md:122]: Please do capitalize search.

search -> Search

**Comment by berndverst** [comment]
@goldwinner you still need to sign the Contributor License Agreement (CLA). See links above.

**Pull request closed without merging**